tensorboard/plugins/hparams/tf_hparams_query_pane/test/tf-hparams-query-pane-test.html

```diff
@@ -204,90 +191,99 @@
         });
 
         test('element updates "sessionGroups" in correct order', function() {
-          // Stop immediate-responding so we can have a server respond
-          // out-of-order.
-          fakeBackendServer.respondImmediately = false;
-          const oldRequests = fakeBackendServer.requests.length;
-          element._queryServer();
-          element._queryServer();
-          // Make sure we have two new requests.
-          assert.equal(fakeBackendServer.requests.length, oldRequests + 2);
+          let resolveFirstQuery = null;
+          let resolveSecondQuery = null;
+          // We set calls #1 and #2 here to return pending promises so
+          // we can control their execution order by the fake backend.
+          // Note that the first call (#0) is already done when the element
+          // initializes.
+          fakeBackend.listSessionGroups.onCall(1).returns(
+              new Promise(resolve => resolveFirstQuery = resolve));
+          fakeBackend.listSessionGroups.onCall(2).returns(
+              new Promise(resolve => resolveSecondQuery = resolve));
+          return element._getExperimentResolved.then(() => {
+            const prevCallCount = fakeBackend.listSessionGroups.callCount;
+            firstQueryDone = element._queryServerNoDebounce();
```

Comment:
You may be forgetting `const` here. It may be defined as let in ancestor scope but regardless, it is a good hygiene to declare one here as const.

Reply:
Thanks! Done.